.github/CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,82 @@
+This code of conduct outlines our expectations for participants within the
+**Nona Creative** orgnaisation and community. We are committed to providing a welcoming and inspiring community for all and expect our code of conduct to be honored.
+
+Our company strives to:
+
+- **Be friendly and patient.**
+- **Be welcoming**: We strive to be a community that welcomes and supports
+  people of all backgrounds and identities. This includes, but is not limited to members of any race, ethnicity, culture, national origin, colour, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+- **Be considerate**: Your work will be used by other people, and you in turn
+  will depend on the work of others. Any decision you take will affect users and colleagues, and you should take those consequences into account when making decisions. Remember that we’re a world-wide community, so you might not be communicating in someone else’s primary language.
```

Comment:
I think we can remove the line about world-wide community and other languages. We all use English as our primary language in the office.